connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/config/EmailConnectorConfig.java

```diff
@@ -0,0 +1,131 @@
+package com.redhat.cloud.notifications.connector.email.config;
+
+import com.redhat.cloud.notifications.connector.ConnectorConfig;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.util.HashMap;
+import java.util.Map;
+
+@ApplicationScoped
+public class EmailConnectorConfig extends ConnectorConfig {
+    private static final String BOP_API_TOKEN = "notifications.connector.user-provider.bop.api_token";
+    private static final String BOP_CLIENT_ID = "notifications.connector.user-provider.bop.client_id";
+    private static final String BOP_ENV = "notifications.connector.user-provider.bop.env";
+    private static final String BOP_URL = "notifications.connector.user-provider.bop.url";
+    private static final String FETCH_USERS_RBAC_ENABLED = "fetch.users.rbac.enabled";
```

Comment:
This doesn't match the env var in the ClowdApp template.
```suggestion
    private static final String FETCH_USERS_RBAC_ENABLED = "notifications.connector.fetch.users.rbac.enabled";
```